src/main/java/org/apache/commons/compress/compressors/zstandard/ZstdUtils.java

```diff
@@ -85,7 +85,7 @@ private static boolean internalIsZstdCompressionAvailable() {
      * <p>This defaults to {@code false} in an OSGi environment and {@code true} otherwise.</p>
      * @param doCache whether to cache the result
      */
-    public static void setCacheZstdAvailablity(final boolean doCache) {
+    public static void setCacheZstdAvailability(final boolean doCache) {
```

Comment:
and again, we will have to live with a misspelled method name.

Reply:
> and again, we will have to live with a misspelled method name.

done.